Tensile/TensileCreateLibrary.py

```diff
@@ -165,7 +165,7 @@ def which(p):
     else:
         exes = [p+x for x in ['', '.exe', '.bat']]
     system_path = os.environ['PATH'].split(os.pathsep)
-    if supportedLinuxCompiler(p) and 'CMAKE_CXX_COMPILER' in os.environ and os.path.isfile(os.environ['CMAKE_CXX_COMPILER']):
+    if supportedCompiler(p) and 'CMAKE_CXX_COMPILER' in os.environ and os.path.isfile(os.environ['CMAKE_CXX_COMPILER']):
         return os.environ['CMAKE_CXX_COMPILER']
```

Comment:
Same comment as in earlier duplicate code, but comparing the which( p ) I see this version has fix so on windows the .bat file is considered first (as simpler to use) but on the other version it considers versions without suffix.  Would think fixing one in Common.py and reusing by import if possible.  Or just copy the exes if from this one.    Strangely this is not which function if it allows a CXX compiler only env return which you are asking for which assembler.